app/parse.py

```diff
@@ -8,5 +13,47 @@ class Course:
     duration: str
 
 
+def parse_single_course(course: BeautifulSoup) -> Course:
+    return Course(
+        name=course.select_one(
+            "h1[data-qa='profession-title']"
+        ).text.split(":")[0],
+        short_description=course.select_one(
+            "p.typography_landingTextMain__Rc8BD."
+            "SalarySection_aboutProfession__1VFHK"
+        ).text,
+        duration=course.select_one(
+            ".ComparisonTable_tableBody__W5hzV.mb-24 "
+            "> div:nth-of-type(7) "
+            "> div:nth-of-type(2)"
+        ).text
```

Comment:
This CSS selector is also dependent on the current HTML structure. If the structure changes, it might not work as expected. Adding error handling would be beneficial.